Nodejs/Product/Nodejs/TypingsAcquisitionTool/bin/install_typings

```diff
@@ -18,19 +18,21 @@ var argv = minimist(process.argv.slice(2), {
 
 var emitter = new events.EventEmitter();
 
+var options = {
+    save: argv.save,
+    emitter: emitter,
+    global: true,
+    cwd: argv.cwd || process.cwd()
+};
+
 var packagesToInstall = argv._;
 
 if (!packagesToInstall.length) {
     // top level package install
     typingsTool.installTypingsForProject(options)
 } else {
     typingsTool.runAll(packagesToInstall.map(function (name) {
-        var options = {
```

Comment:
This won't work as intended. By having options outside of this function body, you are passing the same options object in with every call to installTypingsForPackage, so unless it executes synchronously and doesn't refer to the object again - all later uses refer to the same object - so only the last settings take effect. Safer to just create a different 'options' object (maybe inline) if the other branch of the 'if' statement.